### PR TITLE
Controls pages add minimal data

### PR DESCRIPTION
**Pull request by HankG**
**What kind of change does this PR introduce?**
Adds minimal information to each of the TODO control docs pages

**What is the current behavior?**
No data for most controls

**What is the new behavior?**
Each control type now has a control doc with a minimum of:
* Basic description
* Link to the API documentation
* Link to the GitHub source
Also deleted the obsolete DropDown control

### CODE REVIEW

File: wwwroot/docs/controls/repeatbutton.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 Title: RepeatButton
 ---
-TODO
\ No newline at end of file
+
+The RepeatButton control's full documentation can be found [here](/api/Avalonia.Controls/RepeatButton/){target="_blank"}.
```
Comment (jmacato) [inline wwwroot/docs/controls/repeatbutton.md]: Nit, missing apostrophes on RepeatButton

File: wwwroot/docs/controls/separator.md [hunk 2]
```diff
@@ -1 +1,7 @@
-TODO
\ No newline at end of file
+Title: Separator
+---
+
+The Separator control's full documentation can be found [here](/api/Avalonia.Controls/Separator/).
```
Comment (jmacato) [inline wwwroot/docs/controls/separator.md]: and here too

File: wwwroot/docs/controls/repeatbutton.md (hunk 1 above)
Comment (HankG) [reply wwwroot/docs/controls/repeatbutton.md]: I was kind of thinking about it as a proper noun thing but I didn't capitalize it as such either.  So it's like "the control's full documentation..." (which control?) "The RepeatButton Control's full documentation.  It should really be the escaped text in a description of the control but I couldn't figure out what it was from other documentation sources or the source of the control itself.  If you have a good description of it I'd rather replace it with that like the others.

File: wwwroot/docs/controls/separator.md (hunk 2 above)
Comment (HankG) [reply wwwroot/docs/controls/separator.md]: Same as above however this class is empty, I was wondering if this should be deleted.

File: wwwroot/docs/controls/repeatbutton.md (hunk 1 above)
Comment (HankG) [reply wwwroot/docs/controls/repeatbutton.md]: ...just to capture here from Gitter...you wanted backticks not apostrophes...yep will do...

File: wwwroot/docs/controls/separator.md (hunk 2 above)
Comment (HankG) [reply wwwroot/docs/controls/separator.md]: ...just to capture here from Gitter...you wanted backticks not apostrophes...yep will do...

File: wwwroot/docs/controls/repeatbutton.md (hunk 1 above)
Comment (jmacato) [reply wwwroot/docs/controls/repeatbutton.md]: yep, kept calling them apostrophes :laughing:

Comment (HankG) [reply wwwroot/docs/controls/repeatbutton.md]: I believe I've addressed this with the latest commits.

File: wwwroot/docs/controls/separator.md (hunk 2 above)
Comment (HankG) [reply wwwroot/docs/controls/separator.md]: I believe I've addressed this with the latest commits.

**Pull request merged**